### DISCUSSION TITLE
Skywalking with Jboss

### DISCUSSION THREAD

**Discussion by DuraiDhyaan** [apache/skywalking#11874] [Q&A]
Can Skywalk supports Jboss EAP .If so share the steps to configure.

**Comment by wu-sheng** [comment]
Don't keep putting duplicate information. OSS discussion is an async channel. Few people keep eyes on discussion like their duty.

**Comment by wu-sheng** [comment]
Duplicated with https://github.com/apache/skywalking/discussions/11871
